src/components/existing-request/existing-request-display.vue

```diff
@@ -781,10 +783,20 @@ export default class ExistingRequestDisplay extends Mixins(
     return this.affiliateOrLogin()
   }
 
+  async amalgamateYourBusiness (): Promise<void> {
+    // safety check
+    if (!this.isNrApprovedOrConditional) return
+
+    return this.affiliateOrLogin()
+  }
+
   /** Affiliates the current NR if authenticated, or prompts login if unauthenticated. */
   private async affiliateOrLogin (): Promise<any> {
     if (this.isAuthenticated) {
       await this.createAffiliation(this.nr)
+      if (isAmalgamation) {
+        await this.amalgamateNow(this.nr.entity_type_cd)
+      }
```

Comment:
Doesn't createAffiliation also handle the "amalgamate now" functionality?

Why do you need amalgamateNow() here but there is no corresponding incorporateNow() -- how doesn't "incorporate now" work and why aren't you doing the same thing?

Reply:
It's in the same file but it isn't called in createAffiliation. Should I handle the if statement there instead to call amalgamateNow?

Reply:
1. Line 797 shows up as a code error when I look at it in VS Code. Please fix it.

2. Since line 797 isn't working, how were you able to test this? Please make a best effort to test things before asking reviewers to spend their time on it. Otherwise, ask for help, or call out in the PR that you're not sure if this is the way to do something or you aren't able to test or something.

3. If I work down through the code...
```
existing-name-request.vue::affiliateOrLogin()
  |
  +-> nr-afficreateAffiliation()
    |
    +-> createBusiness()
      |
      +-> getBusinessRequest()
```
... then I don't see any handling for an amalgamation business request.

Granted, there seems to be some existing code duplication between createBusiness() and createBusinessIA() but let's ignore that for now.

Just do the same thing as for incorporation applications (except update the business request for an amalgamation filing).